riotfile.py

```diff
@@ -302,10 +302,18 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
             command="pytest {cmdargs} tests/contrib/pynamodb",
             venvs=[
                 Venv(
-                    pys=select_pys(),
+                    pys=select_pys(min_version=3.5),
+                    pkgs={
+                        "pynamodb": [">=4.0,<4.1", ">=4.1,<4.2", ">=4.2,<4.3", ">=4.3,<4.4", latest],
+                        "moto": ">=1.0,<2.0",
```

Comment:
I wonder if you can restructure like:

```python
Venv(
  name="pynamodb",
  command="...",
  pkgs={
    "pynamodb": [">=4.0,<4.1", ">=4.1,<4.2", ">=4.2,<4.3", ">=4.3,<4.4", latest],
    "moto": ">=1.0,<2.0",
  },
  venvs=[
    Venv(pys=select_pys(min_version=3.5)),
    Venv(pys=["2.7"], pkgs={"rsa": "<4.7.1"}),
  ]
)
```

Reply:
Good point, that's what we had in mind with the syntax!